spec/api-server.spec.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+describe("ApiServer", () => {
+    var apiServer, ApiServer, config, webServerMock;
```

Comment:
Only `ApiServer` and `webServerMock` should be here.  The rest are not touched in the `beforeEach()` and thus won't need to be in scope yet.